File: 1. Beginner/3145/3145.ex
input = IO.gets("")
[n_str, x_str] = String.split(String.trim(input))

{n, _} = Float.parse(n_str)
{x, _} = Float.parse(x_str)

IO.puts(:erlang.float_to_binary(x / (n + 2), decimals: 2))

File: 1. Beginner/3145/3145.dart
import 'dart:io';

void main() {
  String? input = stdin.readLineSync();
  if (input != null) {
    List<String> values = input.split(' ');
    double n = double.parse(values[0]);
    double x = double.parse(values[1]);
    print("${(x / (n + 2)).toStringAsFixed(2)}");
  }
}